### PR TITLE
feat(RFC): File RFC-0002 - deno

### PR DESCRIPTION
**Pull request by nothingismagick**
Please consult the changefile for the content.

### CODE REVIEW

**Comment by altitudems** [comment]
Looks amazing :)

**Comment by jbolda** [comment]
Seems like this isn't contentious and just need the effort to carry it over the finish line. I presume we can probably merge this?

File: texts/002-deno.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+## Motivation
+Deno provides a complete API that is comparable to node. If we can have a light version of deno ~ 10 mb, and a tauri webview / api ~ 2 mb, we can have an electron style binary that is smaller than 12mb without compression -- (comparable to 130mb for the MacOS electron framework)
+
+## User Benefit
+The user may build a tauri app with only javascript and without the need to install anything else. They can build their app by downloading the tauri binary and writing some javascript code (using deno v8) following an Electron-like development pipeline. 
+
+The user gains full access to the Deno Namespace/API and we don't have to maintain a complete API as a result. 
+
+The binary is built with deno permissions are disabled by default. The user may enable the appropriate permissions giving them access to the file system, network settings and the environment. Access to security sensitive areas or functions require the use of permissions which need to be explicitly granted to the process. (https://deno.land/manual/getting_started/permissions#permissions)
```
Comment (nklayman) [inline texts/002-deno.md:22]:
```suggestion
The binary is built with all [deno permissions](https://deno.land/manual/getting_started/permissions) disabled by default. The user may enable the appropriate permissions giving them access to the file system, network settings and the environment. Access to security sensitive areas or functions require the use of permissions which need to be explicitly granted to the process. (https://deno.land/manual/getting_started/permissions#permissions)
```

**Comment by nklayman** [comment]
Looks good to me, just the one fix I suggested.

**Comment by nothingismagick** [comment]
Closing this RFC because it is very, very stale and never followed up upon.

**Pull request closed without merging**